hir2/src/ir/operation.rs

```diff
@@ -458,7 +458,10 @@ impl Operation {
             self.set_attribute(attr_name, Some(attr));
         }
 
-        symbol.borrow_mut().insert_use(user);
+        // TODO: restore after https://github.com/0xPolygonMiden/compiler/issues/403
+        // is fixed
+        //
+        // symbol.borrow_mut().insert_use(user);
```

Comment:
@bitwalker I have not found a quick solution for #403 so I temporarily commented it out.